### PR TITLE
Change docs order

### PR DESCRIPTION
**Pull request by raminqaf**
### Home page
Screenshot 2023-02-22 at 15 36 55

### The user-guide section
Screenshot 2023-02-22 at 15 37 08

### CODE REVIEW

**Review by sujuka99 — COMMENTED** [review]
It looks good, but I dislike that "Examples" in the nav-bar behaves differently from the rest of the titles. I would say either none of them or all of them should be clickable even if the text on some is minimal.

**Review by sujuka99 — APPROVED** [review]
LGTM

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -62,18 +62,17 @@ markdown_extensions:
...
-      - user/getting-started/index.md
       - user/getting-started/setup.md
       - user/getting-started/quick-start.md
       - user/getting-started/teardown.md
     - Examples:
-      - user/examples/index.md
       - user/examples/atm-fraud-pipeline.md
     - References:
-      - user/references/index.md
+      - user/references/components.md
```
Comment (disrupted) [inline docs/mkdocs.yml:76]: doesn't this belong to https://github.com/bakdata/kpops/pull/112?

Comment (raminqaf) [reply docs/mkdocs.yml:76]: It is! This will be merged into @sujuka99 branch. Have a look at the top of the PR. The destination branch is not `main`.

**Pull request merged**